README.md

```diff
@@ -19,14 +19,29 @@ gem 'mysql_framework'
 
 ### Environment Variables
 
+#### MySQL Connection Variables
+
 * `MYSQL_HOST` - MySQL Host
 * `MYSQL_PORT` - MySQL Port
 * `MYSQL_DATABASE` - MySQL database name
 * `MYSQL_USERNAME` - MySQL username
 * `MYSQL_PASSWORD` - MySQL password
-* `MYSQL_PARTITIONS` - The number of partitions where a table is partitioned (default: `500`)
+
+#### MySQL Connection Pooling Variables
+
+* `MYSQL_START_POOL_SIZE` - how many connections should be created by default (default: `1`)
+* `MYSQL_MAX_POOL_SIZE` - how many connections should the pool be allowed to grow to (default: `5`)
+
+#### MySQL Migration Variables
+
+* `MYSQL_MIGRATION_TABLE` - the name of the table that holds a record of applied migrations (default: `migration_script_history`)
+* `MYSQL_MIGRATION_LOCK_TTL` - how long the tables should be locked for whilst performing migrations (default: `2000` / `2 seconds`)
 * `REDIS_URL` - The URL for redis - used for managing locks for DB migrations
 
+#### Miscellaneous Variables
+
+* `MYSQL_PARTITIONS` - The number of partitions where a table is partitioned (default: `500`)
```

Comment:
This phrase is a bit confusing.
Do you mean 'The number of partitions when a table is partitioned'?